docker-compose-generator/docker-fragments/opt-add-balanceofsatoshis.yml

```diff
@@ -0,0 +1,14 @@
+version: "3"
+services:
+  bos:
+    container_name: bos
+    image: "alexbosworth/balanceofsatoshis:latest"
+    restart: unless-stopped
+    volumes:
+      - "lnd_bitcoin_datadir:/home/node/.lnd:ro"
```

Comment:
I thought about that too, but it seems like BOS expects a node name then too `~/.bos/YOUR_NODE_NAME/credentials.json`). 

Maybe it'd be easiest to symlink it then. Might be good in general, because with other intergrations we had this problem before too. I think the location of the admin.macaroon in our setup is for backwards compatibility — maybe there's a chance here to find a way forward with the nowadays default location :)